src/Symfony/Component/Security/Core/Validator/Constraints/UserPasswordValidator.php

```diff
@@ -39,6 +39,10 @@ public function validate($password, Constraint $constraint)
             throw new UnexpectedTypeException($constraint, __NAMESPACE__.'\UserPassword');
         }
 
+        if (null === $password || '' === $password) {
+            return;
```

Comment:
This change breaks FOSUserBundle's `current_password` field used in `ChangePasswordFormType` and `ProfileFormType` allowing to send empty passwords instead of the user's current one.